docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+volumes:
+  nextcloud:
+  db:
+
+services:
+
+  db:
+     image: mariadb:10.6
+     restart: always
+     command: --transaction-isolation=READ-COMMITTED --log-bin=binlog --binlog-format=ROW
+     volumes:
+       - db:/var/lib/mysql
+     environment:
+       - MYSQL_ROOT_PASSWORD=root
+       - MYSQL_PASSWORD=sqlusr
+       - MYSQL_DATABASE=nextcloud
+       - MYSQL_USER=nextcloud
+
+  app:
+    image: nextcloud
+    restart: always
+    ports:
+      - 8080:80
+    links:
+      - db
+    volumes:
+      - nextcloud:/var/www/html
+    environment:
+      - MYSQL_PASSWORD=sqlusr
+      - MYSQL_DATABASE=nextcloud
+      - MYSQL_USER=nextcloud
+      - MYSQL_HOST=db
+      - NEXTCLOUD_ADMIN_USER:testuser
+      - NEXTCLOUD_ADMIN_PASSWORD:1234
+      - NEXTCLOUD_TRUSTED_DOMAINS:localhost:8080 172.27.192.1:8080
```

Comment:
this throws me the following error:
``ERROR: environment variable name 'NEXTCLOUD_TRUSTED_DOMAINS:localhost:8080 172.27.192.1:8080' may not contain whitespace.`` Did you mean to rremove the address at the end?

Reply:
we can leave that out, since i don't need the second address anymore (was a wsl2 thing) and by default, localhost is trusted, but this should have actually worked: 

One or more trusted domains can be set through environment variable, too. They will be added to the configuration after install.

    NEXTCLOUD_TRUSTED_DOMAINS (not set by default) Optional space-separated list of domains
    
It could also be, that this is still bugged.

Reply:
did it now. I checked again and it's strange. The automatic login/registration for the admin doesn't seem to work, I'm not sure now if it even worked beforehand. Otherwise the rest should be fine now.